.codecov.yml

```diff
@@ -0,0 +1,12 @@
+comment: off
+github_checks:
+  annotations: false
```

Comment:
why don't we want them?

Reply:
Too much noise in the PR? We can go to the report from the check. If you guys think is helpful, we can keep them.

Reply:
> Too much noise in the PR?

Is there an example to look at (the codecov docs example is quite small)? Leaving annotations off is probably the way to go, but I'm curious to see how they look in a PR.

Reply:
You have an example above in this PR, this is [the link](https://github.com/OpenZeppelin/cairo-contracts/pull/494#issuecomment-1290491461)

Reply:
It's missing the base commit in the codecov link, so it can't compare :( 

Haha it's not a big deal

Reply:
This is for the comment in the PR itself, following the codecov link is not required to check it right?

Reply:
`annotations` and `comment` are different things.

Annotations places inline notes in the "Files changed" tab in a PR that say "this line is not covered". They can be useful but also noisy. You may want to try them and disable later.